book/src/sec-4-wql.md

```diff
@@ -51,25 +51,25 @@ Updates the content by replacing the previous entity map in entity tree key `my_
 
 ### UPDATE CONTENT
 
-Updates the content by numerical addition or string concatenation of the previous entity map in entity tree key `my_entity_name` with the entity id `48c7640e-9287-468a-a07c-2fb00da5eaed`. Non numerical or non string value will just be replaced. If key doesn't  exist it will be created.
+Updates the content by numerical addition or string concatenation of the previous entity map in entity tree key `my_entity_name` with the entity id `48c7640e-9287-468a-a07c-2fb00da5eaed`. Non numerical or non string value will just be replaced. If the key doesn't exist it will be created.
 
 * `UPDATE my_entity_name CONTENT {a: -4, b: 32,} INTO 48c7640e-9287-468a-a07c-2fb00da5eaed` this will add `-4` to entity map key `a` and add `32` to entity map key `b` in the current entity map stored in entity id `48c7640e-9287-468a-a07c-2fb00da5eaed`.
 
 ### MATCH UPDATE
 
-Similar to SET, but it requires a pre-condition to be satisfied.
+Similar to SET, but it requires a precondition to be satisfied.
 
 * `MATCH ALL(a == 1, b >= 3, c != \"hello\", d < 7,) UPDATE this_entity SET {a: 123, g: NiL,} INTO d6ca73c0-41ff-4975-8a60-fc4a061ce536` if all conditions defined inside `ALL` are satisfied the set update will happen.
     - `ALL` is an logical `and` between all conditions, meaning that all of them must be true.
     - `ANY` is an logical `or` between all conditions, meaning that at least one of them must be true.
     - **NULL KEYS**, `ALL` returns an error if a null key is present and `ANY` just ignores null keys.
     - Possible conditions are:
-        - `==` means equals, so if `a == 100`, this means that the entity map key `a` must equals to `100`.
-        - `!=` means not equals, so if `a != 100`, this means that the entity map key `a` must not equals to `100`.
-        - `>=` means greater or equal, so if `a >= 100`, this means that the entity map key `a` must br greater or equals to `100`.
-        - `<=` means lesser or equal, so if `a <= 100`, this means that the entity map key `a` must be lesser or equals to `100`.
-        - `>` means greater, so if `a > 100`, this means that the entity map key `a` must be greater than `100`. 
-        - `<` means lesser, so if `a < 100`, this means that the entity map key `a` must be lesser than `100`.
+        - `==` means equals, so if `a == 100`, this means that the entity map key `a` must equal to `100`.
```

Comment:
Again, equals is equivalent to the verb, so a must equals to x is correct, and I prefer. While this change was correct `!=` means not equal, so if `a != 100` because it refers to the mathematics operation.

Reply:
It shouldn't have the `s` because the equality is being referred to the key, which is singular.

I think the `equals` in the beginning of the sentence is the one that describes what you've said correctly, that's why it has the `s`.